CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ different versioning scheme, following the Haskell community's
     * Clang 3.4 or newer
     * GNU C++ 4.7 or newer
     * Microsoft Visual C++ 2013 or newer
+* The `bond::Apply` now has a uniform signature. Users who were calling
+  the overload for `Marshaler<Writer>` transform and were _mistakenly_
+  passing `Writer` explicitly (i.e. `bond::Apply<Writer>(marshaler, value)`)
+  they will get a compile error. To fix, please remove the `<Writer>` part.
```

Comment:
Here's a copy edited version:

The `bond::Apply` function now has a uniform signature. Call sites for 
the `Marshaler<Writer>` transform overload that were _mistakenly_
passing `Writer` explicitly (e.g. `bond::Apply<Writer>(marshaler, value)`)
will now get a compiler error. To fix, remove the `<Writer>` part:
`bond::Apply(marshaler, value)`.